docs/manta/DevelopOnDolphin.md

```diff
@@ -0,0 +1,99 @@
+# 🛠 Develop on Dolphin
```

Comment:
I'm not sure both docs should be added for manta for now.

Reply:
Yeah I think we should just remove the Manta SDK and Develop on Manta sections for now.

Reply:
Im removing the SDK section on Manta for now, seeing as there is no Manta parachain yet.